db/migrate/20241020174649_add_speakerdeck_to_speaker.rb

```diff
@@ -0,0 +1,5 @@
+class AddSpeakerdeckToSpeaker < ActiveRecord::Migration[8.0]
+  def change
+    add_column :speakers, :speakerdeck, :string
```

Comment:
I know it it controversial but I prefer for most cases to default string to "" with null: false

Reply:
👍🏼